tools/add_to.hpp

```diff
@@ -140,52 +140,51 @@ namespace antler {
       }
 
       add_to_project(CLI::App& app) {
-         path = system::fs::current_path().string();
          subcommand = app.add_subcommand("add", "Add an app, dependency, library or test to your project.");
-         subcommand->add_option("-p, path", path, "Path containing the project's yaml file.");
+         subcommand->add_option("-p", path, "Path containing the project's yaml file.")->default_val(".");
 
          app_subcommand = subcommand->add_subcommand("app", "Add a new app to your project.");
          app_subcommand->footer(std::string(R"(Examples:)")

```

Comment:
Examples are out of date in this and other files.

Reply:
examples you pointed are still look working. Do you mean to add examples with full argument names?

Reply:
![image](https://user-images.githubusercontent.com/380917/235985339-16d96211-1af3-4218-931b-9a0a698bc0cc.png)

As you can see in the image, commit e8a2e05 has incorrect examples in its output for `antler-proj update app --help`.

The subcommands with bad examples are:
- update app
- update lib
- update dep
- remove app
- remove dep
- remove lib
- populate
- init
- add app
- add dep
- add lib